dpytools/config/README.md

```diff
@@ -0,0 +1,159 @@
+# dpytools: Config
+
+Configuration class to enforce and police python application settings.
+
+## Usage
+
+The from_env() method is used to define a config object and populate it with 
```

Comment:
This may be more of a personal preference, but I like it when methods etc are formatted as code (e.g. `from_env()` instead of from_env()) - I find it easier to distinguish between the actual method names and the rest of the text.

Reply:
Agreed